test/integration/helpers.bash

```diff
@@ -37,17 +37,28 @@ function wait_until_reachable() {
 }
 
 # Start the swarm manager in background.
-function start_manager() {
-	${SWARM_ROOT}/swarm manage -H $SWARM_HOST "$@" `join , ${HOSTS[@]}` &
+function swarm_manage() {
+	${SWARM_ROOT}/swarm manage -H $SWARM_HOST "$@" &
 	SWARM_PID=$!
 	wait_until_reachable $SWARM_HOST
 }
 
+function swarm_join() {
+	for h in ${HOSTS[@]}; do
+		echo "Swarm join $h $@"
+		${SWARM_ROOT}/swarm join --addr=$h "$@" &
+	done
+}
+
 # Stops the manager.
 function stop_manager() {
```

Comment:
For sake of consistency, this should probably be something like `swarm_manage_cleanup`?

Since we are not talking about a manager anymore, maybe it makes sense to simply have a `_cleanup` function